### PR TITLE
LED_BUILTIN should be a #define

### PR DESCRIPTION
**Pull request by d-a-v**
Arduino only defines LED_BUILTIN, should BUILTIN_LED be removed ?

### CODE REVIEW

File: variants/ESPDuino/pins_arduino.h [hunk 1]
```diff
@@ -33,7 +33,7 @@
 static const uint8_t SDA = PIN_WIRE_SDA;
 static const uint8_t SCL = PIN_WIRE_SCL;
 
-static const uint8_t LED_BUILTIN = 16;
-static const uint8_t BUILTIN_LED = 16;
+#define LED_BUILTIN 16
```
Comment (earlephilhower) [inline variants/ESPDuino/pins_arduino.h]: Could a compiler #define such as "-DBUILTIN_LED=LED_BUILTIN" and remove the need for all these repeated 2nd #define BUILTIN_LED X?

Comment (d-a-v) [reply variants/ESPDuino/pins_arduino.h]: BUILTIN_LED is not used in core, but seems to be historically used in esp8266 code outside.
What about moving this define into ```variants/generic/common.h``` (hidden, wrong name less exposed, less loaded gcc cmdline) ?

Comment (devyte) [reply variants/ESPDuino/pins_arduino.h]: If it's not standard, how about putting it somewhere safe and deprecating its use?

Comment (d-a-v) [reply variants/ESPDuino/pins_arduino.h]: Good idea

File: variants/espino/pins_arduino.h [hunk 2]
```diff
@@ -34,15 +34,16 @@
...
 static const uint8_t SCL = PIN_WIRE_SCL;
 
-static const uint8_t LED_BUILTIN = 2;
+#define LED_BUILTIN 2
 static const uint8_t LED_BUILTIN_R = 2;
 static const uint8_t LED_BUILTIN_G = 4;
 static const uint8_t LED_BUILTIN_B = 5;
 
-static const uint8_t BUILTIN_LED = 2;
+/*
```
Comment (devyte) [inline variants/espino/pins_arduino.h]: remove commented code

File: variants/espino/pins_arduino.h [hunk 3]
```diff
@@ -34,15 +34,16 @@
...
 static const uint8_t BUILTIN_LEDR = 2;
 static const uint8_t BUILTIN_LEDG = 4;
 static const uint8_t BUILTIN_LEDB = 5;
 static const uint8_t BUILTIN_BUTTON = 0;
```
Comment (devyte) [inline variants/espino/pins_arduino.h]: maybe there should be a BUTTON_BUILTIN?

Comment (d-a-v) [reply variants/espino/pins_arduino.h]: Right. BUTTON_BUILTIN added, commented BUILTIN_* removed.

**Pull request merged**